Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Users
+    {
+        public int UsersId { get; set; }
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Password { get; set; }
+        public string MembershipType { get; set; }
+
+        public ICollection<ArticlesRead> ArticleRead { get; set; }
```

Comment:
you really got to initialize these here is an example 

` 
User(){
Initialise();
}

public void Initialise()
        {
            ArticleRead = new HashSet<ArticleRead >();you can add all the collections in here and just call initialize in the constructor
        }`